marketplace_builder/views/pages/how-platformos-works/platformos-workflow/codebase.liquid

```diff
@@ -0,0 +1,36 @@
+---
+converter: markdown
+metadata:
+  title: Codebase
+  description: 
+slug: how-platformos-works/platformos-workflow/codebase
+searchable: true
+---
+
+You develop your site (write code) in your **codebase** in your local environment. 
+
+## Required directory structure 
+
+In order to correctly communicate with the platformOS engine and API, your codebase should be organized into a specific directory structure. The root directory of your project should contain the marketplace_builder directory. All directories other than the marketplace_builder will be ignored by the Marketplace Kit when deploying and syncing, so you can keep all your JS and CSS files outside of the marketplace_builder directory, and use any pre-processors you want to automatically generate the end result files in a proper path inside marketplace_builder. 
```

Comment:
The part on ignoring is no longer true since we have `modules` directory as well now

Reply:
Removed it, thank you!